meshroom/ui/qml/main.qml

```diff
@@ -838,11 +838,17 @@ ApplicationWindow {
                 reconstruction: _reconstruction
                 readOnly: _reconstruction.computing
 
-                function viewAttribute(attribute, mouse) {
-                    let viewable = false;
-                    viewable = workspaceView.viewIn2D(attribute);
-                    viewable |= workspaceView.viewIn3D(attribute, mouse);
-                    return viewable;
+                function viewNode(node, mouse) {
+                    // 2D viewer
+                    viewer2D.displayedNode = node;
+
+                    // 3D viewer
+                    for(var i=0; i < node.attributes.count; ++i)
```

Comment:
could be good to add semantic for 3D too

Reply:
I think we should make a separate issue to normalize the use of the _semantic_ field (so we can anticipate problems that may appear on the long term before using this concept in too many places of the code), and at this point yep sure it'll be a good idea to add 3D to semantics. What do you think ?